testdriver/testplan.py

```diff
@@ -71,21 +78,36 @@ def runPlan(self):
     # Check for option to set version of executor
     # TODO
 
+    # If icu_version is "latest" or not set, get the highest numbered
+    # version of the test data
+    input_root = os.path.join(self.options.file_base,
+                              self.options.input_path)
+    icu_test_dirs = glob.glob('icu*', root_dir=input_root)
+    if not icu_test_dirs:
+      raise Exception('No ICU test data found in directory %s' % input_root)
+
+    if self.icu_version not in icu_test_dirs:
+      newest_version = sorted(icu_test_dirs, reverse=True)[0]
```

Comment:
Are we okay to assume that the string representation of the version number can be sorted without changing the correct logical order?

FYI, according to [this SO answer](https://stackoverflow.com/a/2574230), there's a built-in library to sort strings of semantic versions.

Reply:
Good idea to fix this. However, for now the test dirs are labeled icu69, icu70, icu71, so this should be documented. Adding to the code for next push.